DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -1,10 +1,30 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
 
 public class ComputerGeekGame
 {
+    private string _scoreBoardfile = "DeveloperGameLeaderBoard.txt";
+    private List<KeyValuePair<int, string>> _leaderBoard = new List<KeyValuePair<int, string>>();
+
     public ComputerGeekGame()
     {
-        // To do.
+        try
+        {
+            if (File.Exists(_scoreBoardfile))
+            {
+                ReadScoresFromFile();
+            }
+            else
+            {
+                FileStream stream = File.Create(_scoreBoardfile);
+                stream.Close();
+            }
```

Comment:
Are you going to create some placeholder scores if they are playing for the first time and don't yet have a score board? Or if there was nothing in the file?

Reply:
I am not planning on creating any place holder scores.

Reply:
Ok, just keep that in mind they might be empty when using the list of scores